Blossom/README.md

```diff
@@ -0,0 +1,17 @@
+# BLOSSOM
+
+A little theme for your Spotify client.
+
+![image](https://user-images.githubusercontent.com/72624799/203471073-4a5e6cf0-a5dc-4ecc-9a12-56d5fc716ac4.png)
+
+Made for a dark theme look thats pleasing to the eye.
+
+# IMPORTANT
+
+USE SPICETIFY FOR THIS THEME TO WORK!
+
+link to their amazing repo: https://github.com/spicetify/spicetify-cli
+
+And their website: https://spicetify.app/
```

Comment:
You don't need this since its already in the spicetify-themes repo, so you can remove it.

Reply:
Removed!